Sources/UIApplication+handleSDLEvents.swift

```diff
@@ -93,29 +94,29 @@ extension UIApplication {
 
 extension UIApplication {
     static func onWillEnterForeground() {
-        UIApplication.restart {
-            if let runningApplication = UIApplication.shared {
-                runningApplication.delegate?.applicationWillEnterForeground(runningApplication)
-            }
+        #if os(Android)
+        if UIScreen.main == nil { // sometimes we "enter foreground" after just a loss of focus
+            UIScreen.main = UIScreen()
         }
+        #endif
```

Comment:
why is this android only?

Reply:
It's not necessary on Mac.

The fact that it's wrapped in a condition anyway means we *could* do this all the time, but I'd rather keep things working how we expect, based on our intentions. We are killing UIScreen when enteringBackground on Android only - see below in this file - so it only makes sense to expect to have to recreate it on Android.